### ISSUE TITLE
Many Webhooks (Improvements)

### ISSUE THREAD

**Issue by Master-Antonio** [amargon/nodebb-plugin-discord-notification#2]
Hello amargon.
A serious lack of this plugin is that we cant add more Webhooks.
Because, example, i have several servers where i would like notify my users of the news of the forum.
You can use a arraylist to do this.

var array = [];
array.push();
array,pop();

**Comment by amargon** [comment]
Ah, that’s a good idea.
Another use case could be to send notifications from different categories to different channels.

The tricky part is not the data storage but good UI and an efficient way to determine which webhook to fire when a new post is made. It’s a lot of work and unfortunately I can’t spend so much time on this in the near future.

PRs are welcome though :)

**Comment by TobiTenno** [comment]
This would be fantastic.

**Comment by melt7777** [comment]
Hello! Great plugin and thank you for your work thus far.

Yes, I am also facing a limitation that would be served well with this feature request. I have all public categories notifying to an #announcements channel. I also have a need to add one category to another channel (in this case, for the beta testing team's forum and discord.) 

TLDR: We would like to add a 2nd webhook with its own categories.

Thank you very much for any continued work on this, and hope that you will keep maintaining it to be compatible with nodebb as updates happen. (We know that's not the most fun work!)

Thank you,
~ melt
getpimp.org | miner.farm
